src/coreComponents/physicsSolvers/multiphysics/PoromechanicsSolver.hpp

```diff
@@ -0,0 +1,270 @@
+                                                        array1d< real64 > & s )
+  {
+    // s denotes averageMeanTotalStressIncrement
+    integer i = 0;
+    SolverBase::forDiscretizationOnMeshTargets( domain.getMeshBodies(), [&]( string const &,
+                                                                             MeshLevel & mesh,
+                                                                             arrayView1d< string const > const & regionNames ) {
+      mesh.getElemManager().forElementSubRegions< CellElementSubRegion >( regionNames, [&]( localIndex const,
+                                                                                            auto & subRegion ) {
+        // get the solid model (to access stress increment)
+        string const solidName = subRegion.template getReference< string >( "porousMaterialNames" );
+        geos::constitutive::CoupledSolidBase & solid = SolverBase::getConstitutiveModel< geos::constitutive::CoupledSolidBase >(
+          subRegion, solidName );
+        auto & porosityModel = dynamic_cast< geos::constitutive::BiotPorosity const & >( solid.getBasePorosityModel());
+        arrayView1d< real64 > const & averageMeanTotalStressIncrement_k = solid.getAverageMeanTotalStressIncrement_k();
+        for( localIndex k = 0; k < localIndex( averageMeanTotalStressIncrement_k.size()); k++ )
+        {
+          porosityModel.updateAverageMeanTotalStressIncrement( k, s[i] );
+          i++;
+        }
+      } );
+    } );
+  }
+
+  array1d< real64 > addTwoVecs( const array1d< real64 > & vec1,
+                                const array1d< real64 > & vec2,
+                                const real64 sign )
+  {
+    GEOS_ASSERT( vec1.size() == vec2.size());
+    array1d< real64 > result;
+    const localIndex N = vec1.size();
+    for( localIndex i = 0; i < N; i++ )
+    {
+      result.emplace_back( vec1[i] + sign * vec2[i] );
+    }
+    return result;
+  }
+
+  array1d< real64 > scalarMultiplyAVec( const array1d< real64 > & vec,
+                                        const real64 scalarMult )
+  {
+    array1d< real64 > result;
+    const localIndex N = vec.size();
+    for( localIndex i = 0; i < N; i++ )
+    {
+      result.emplace_back( scalarMult * vec[i] );
+    }
+    return result;
+  }
+
+  real64 dotTwoVecs( const array1d< real64 > & vec1,
+                     const array1d< real64 > & vec2 )
+  {
+    GEOS_ASSERT( vec1.size() == vec2.size());
+    real64 result = 0;
+    const localIndex N = vec1.size();
+    for( localIndex i = 0; i < N; i++ )
+    {
+      result += vec1[i] * vec2[i];
+    }
+    return result;
+  }
+
+  real64 computeAitkenRelaxationFactor( const array1d< real64 > & s0,
+                                        const array1d< real64 > & s1,
+                                        const array1d< real64 > & s1_tilde,
+                                        const array1d< real64 > & s2_tilde,
+                                        const real64 omega0 )
```

Comment:
```suggestion
  real64 computeAitkenRelaxationFactor( array1d< real64 > const & s0,
                                        array1d< real64 > const & s1,
                                        array1d< real64 > const & s1_tilde,
                                        array1d< real64 > const & s2_tilde,
                                        real64 const omega0 )
```
to follow the GEOS convention and everywhere in this file

Reply:
done